adds_functions.js

```diff
@@ -16,5 +16,11 @@ module.exports = {
   frankSubtractTwoNumbers: function(numberOne, numberTwo){
     return numberOne - numberTwo;
   },
-  
+
+  samsAddTwoNumbers: new Function('e', 't', 
```

Comment:
Hahahahah! -- This has to be the most ridiculous thing I have ever seen!

Reply:
Haha, it's so rare to get an opportunity to pull out the weird stuff. It was a good reminder to just have fun and play!